styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -20,22 +20,36 @@
 
 package com.spotify.styx.flyte;
 
+import static com.spotify.styx.state.RunState.MISSING_DEPS_EXIT_CODE;
+import static com.spotify.styx.state.RunState.UNKNOWN_ERROR_EXIT_CODE;
```

Comment:
So that you can use it bellow
```suggestion
import static com.spotify.styx.state.RunState.SUCCESS_EXIT_CODE;
import static com.spotify.styx.state.RunState.UNKNOWN_ERROR_EXIT_CODE;
```

Reply:
right 👍 missed that